src/Databases/DatabaseReplicated.cpp

```diff
@@ -120,7 +141,7 @@ ClusterPtr DatabaseReplicated::getCluster() const
         hosts = zookeeper->getChildren(zookeeper_path + "/replicas", &stat);
         if (hosts.empty())
             throw Exception(ErrorCodes::LOGICAL_ERROR, "No hosts found");
-        Int32 cver = stat.cversion;
+        Int32 cversion = stat.cversion;
```

Comment:
Is there any useful information inside `/replicas` node itself (without children)? Maybe better to check `version ` in addition to `cversion`?

Reply:
As for now `/replicas` node does not store any informatin